### DISCUSSION TITLE
Can i edit a MySQL database just like a spreadsheet via NocoDB ? [Apr 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1890] [Discord Threads]
> 🙋‍♂️ **Thunder**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Yes, absolutely.   
####    
#### If you make an edit - the data gets saved back to database.    
####    
#### - It works like a spreadsheet.    
#### - You can invite your team with various access control   
#### - You can access your db-spreadsheet with NocoDB's API (apis.nocodb.com)  
#  
> 🙋‍♂️ **Thunder**  
#### Awesome thank you. Have a great day  
#  
> 🙋‍♂️ **smart1ne2wo**  
#### how to make change and save to database?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/965337038259900486)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
